spec/services/resource_recommendation_spec.rb

```diff
@@ -0,0 +1,55 @@
+
+describe ResourceRecommendation do
+    subject(:moment) {FactoryBot.build(:moment,
+                                       name: "Test ADDICTION",
+                                       why: "More testing content self-care.",
+                                       fix: "Text tested is @Teachers!!") }
+     
+    let(:recommended_resources) { ResourceRecommendation.new(moment).resources }
```

Comment:
Same here. `recommended_resources` should use `subject` and `moment` use `let`

```suggestion
    subject(:recommended_resources) { ResourceRecommendation.new(moment).resources }
```